### ISSUE TITLE
PEP-8 Python Coding Style?

### ISSUE THREAD

**Issue by kjcole** [berteh/ScribusGenerator#124]
I was thinking about digging into the code, and still often work on paper. Lines that are 170+ characters long in the code are a bit much, as well as other style issues.  So, no printouts for me just yet. See [PEP 8 -- Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/). **IF** I get serious enough about it I'll try to bring it into closer compliance, but, at least going forward, some thought towards following the style guide should be considered. (I don't follow it religiously / obsessively, but many of the recommendations seem sane to me.)

**Comment by berteh** [comment]
You're right. Let me try to improve this a bit, and feel free improve some more or give me tips... as I'm not a great Python coder
